man/systemd.network.xml

```diff
@@ -1009,7 +1009,8 @@
         <varlistentry>
           <term><varname>IPProtocol=</varname></term>
           <listitem>
-            <para>Specifies the protocol to match in forwarding information base (FIB) rules. Accepted values are tcp, udp and sctp.
+            <para>Specifies the IP protocol to match in forwarding information base (FIB) rules. Takes IP protocol number or IP protocol name such as <literal>tcp</literal>,
```

Comment:
hmm, what's an IP protocol number though? maybe add an example in brackets: `(6 for <constant>IPPROTO_TCP</constant>, 17 for <constant>IPPROTO_UDP</constant>, …)` or so.